tests/unit/QuantType.spec.js

```diff
@@ -290,9 +290,46 @@ describe('QuantType.vue', () => {
     })
 
     it('must have the correct conversion factor', () => {
-      expect(quantType.conversionFactor).toEqual('1.000')
+      expect(quantType.conversionFactor).toEqual('1')
     })
 
   })
 
+  describe('Duplex Seq Library', () => {
+
+    beforeEach(() => {
+      cmp = Vue.extend(QuantType)
+      quantType = new cmp({propsData: { quantType: 'duplexSeqLibrary'}})
+    })
+
+    it('must have the correct units', () => {
+      expect(quantType.qcResults.units).toEqual('ng/ul')
+    })
+
+    it('must have the correct conversion factor', () => {
+      expect(quantType.conversionFactor).toEqual('1')
+    })
+
+  })
+
+  describe('Duplex Seq AL Lib', () => {
+
+    beforeEach(() => {
+      cmp = Vue.extend(QuantType)
+      quantType = new cmp({propsData: { quantType: 'duplexSeqALLib'}})
+    })
+
+    it('must have the correct options', () => {
+      expect(quantType.$data).toEqual(quantTypes["duplexSeqALLib"])
+    })
+
+    it('must have the correct units', () => {
+      expect(quantType.qcResults.units).toEqual('nM')
+    })
+
+    it('has some metadata', () => {
+      expect(quantType.hasMetadata()).toBeTruthy()
+    })
+
+  })
```

Comment:
Could add tests throughout this file testing the decimalPlaces for each quant type, now that is added as config. So the regualar normalisation, RNAStock etc would have 1, duplexSeqALLib 20 and the qpcr ones 3 etc

Reply:
Thanks. I added 3, to cover different scenarios - thought one for each would be overkill as they'd be testing the same thing